apis/airtable/query builder/query record count setter/1b357c49-e7a8-4610-bfd4-7260c481d9d4.json

```diff
@@ -0,0 +1,78 @@
+{
+  "implementations": {
+    "cranq": {
+      "nodes": {
+        "110e4226-b428-4404-a7ae-d0d1899aa901": {
+          "coordinates": [
+            1,
+            29
+          ],
+          "name": "item setter",
+          "prototype": "3966e767-b7a7-4221-961d-338a13d18c57",
+          "params": {
+            "922a1964-91d4-4ad3-a9e4-7d245662a978": "\"maxRecords\""
+          }
+        }
+      },
+      "connections": {},
+      "forwards": {
+        "inputs": {
+          "81dceb3c-3bf5-4e2b-99f9-b30e4a97a2d0": {
+            "110e4226-b428-4404-a7ae-d0d1899aa901": {
+              "96243772-f5b9-49b6-b60f-6841b5c8a535": {}
+            }
+          },
+          "50f5307f-27a3-4be9-87db-39041e183708": {
+            "110e4226-b428-4404-a7ae-d0d1899aa901": {
+              "0ab0c723-793b-48a1-87d5-d64267665bd3": {}
+            }
+          }
+        },
+        "outputs": {
+          "e055c747-523c-495e-b6cb-c4fddc083f42": {
+            "110e4226-b428-4404-a7ae-d0d1899aa901": {
+              "7adbf554-0042-42c0-90b4-03b5d1406a88": {}
+            }
+          }
+        }
+      }
+    }
+  },
+  "interface": {
+    "inputs": {
+      "81dceb3c-3bf5-4e2b-99f9-b30e4a97a2d0": {
+        "type": "{\n  optional \"fields\": string[],\n   optional \"filterByFormula\": string,\n   optional \"sort\": \n    {\n      \"field\": string,\n      \"direction\": (\"desc\" or \"asc\")\n    }[],\n    optional \"maxRecords\": number,\n    optional \"pageSize\": number,\n    optional \"view\": string,\n    optional \"returnFieldsByFieldId\": boolean\n}",
+        "name": "query",
+        "description": "Receives the AirTable query to extend. To start a new one, set `{}`."
+      },
+      "50f5307f-27a3-4be9-87db-39041e183708": {
+        "type": "number",
+        "name": "record count",
+        "description": "Receives the number of records to return."
+      }
+    },
+    "outputs": {
+      "e055c747-523c-495e-b6cb-c4fddc083f42": {
+        "type": "{\n  optional \"fields\": string[],\n   optional \"filterByFormula\": string,\n   optional \"sort\": \n    {\n      \"field\": string,\n      \"direction\": (\"desc\" or \"asc\")\n    }[],\n    optional \"maxRecords\": number,\n    optional \"pageSize\": number,\n    optional \"view\": string,\n    optional \"returnFieldsByFieldId\": boolean\n}",
```

Comment:
Wouldn't be simpler to refer to the `query` input port's type here?  Like typeof `query`?
Generally for all builder prototypes...

Reply:
Changed to more specifics instead